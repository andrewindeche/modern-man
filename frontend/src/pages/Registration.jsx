import React from 'react';
const Registration = () => {
    return(
        <div className="registrationForm">
            <form>
            <p>Registration: Create User Account</p>
                <span className="nameLabel">
                    <div className="input-group">
                        <label>Full Name</label>
                        <input type="text" name="fullName" placeholder="Enter your Name" required />
                    </div>
                    <div className="input-group">
                        <label>User Name</label>
                        <input type="text" name="userName" placeholder="Enter your User Name" required />
                    </div>
                </span>
                <span className="contactLabel">
                    <div className="input-group">
                        <label>Email</label>
                        <input type="text" name="email" placeholder="Enter your Email" required />
                    </div>
                    <div className="input-group">
                        <label>Phone Number</label>
                        <input type="text" name="phoneNumber" placeholder="Enter your Phone Number" required />
                    </div>
                </span>
                <span className="passwordLabel">
                    <div className="input-group">
                        <label>Password</label>
                        <input type="text" name="password" placeholder="Enter your Password" required />
                    </div>
                    <div className="input-group">
                        <label>Confirm Password</label>
                        <input type="text" name="confirmPassword" placeholder="Confirm your Password" required />
                    </div>
                </span>
                    <button type="submit">Register</button>
            </form>
        </div>
    )
}

export default Registration;
