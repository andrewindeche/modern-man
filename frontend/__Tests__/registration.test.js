import React from 'react';
import Registration from 'Registration.jsx';
import renderer from 'react-test-renderer';


describe("registration page",() =>{
  it('renders the registration page', () => {
    const tree = renderer.create(<Registration />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})