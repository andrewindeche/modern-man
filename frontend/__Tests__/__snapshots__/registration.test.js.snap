// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`registration page renders the registration page 1`] = `
<div
  className="registrationForm"
>
  <form>
    <p>
      Registration: Create User Account
    </p>
    <span
      className="nameLabel"
    >
      <div
        className="input-group"
      >
        <label>
          Full Name
        </label>
        <input
          name="fullName"
          placeholder="Enter your Name"
          required={true}
          type="text"
        />
      </div>
      <div
        className="input-group"
      >
        <label>
          User Name
        </label>
        <input
          name="userName"
          placeholder="Enter your User Name"
          required={true}
          type="text"
        />
      </div>
    </span>
    <span
      className="contactLabel"
    >
      <div
        className="input-group"
      >
        <label>
          Email
        </label>
        <input
          name="email"
          placeholder="Enter your Email"
          required={true}
          type="text"
        />
      </div>
      <div
        className="input-group"
      >
        <label>
          Phone Number
        </label>
        <input
          name="phoneNumber"
          placeholder="Enter your Phone Number"
          required={true}
          type="text"
        />
      </div>
    </span>
    <span
      className="passwordLabel"
    >
      <div
        className="input-group"
      >
        <label>
          Password
        </label>
        <input
          name="password"
          placeholder="Enter your Password"
          required={true}
          type="text"
        />
      </div>
      <div
        className="input-group"
      >
        <label>
          Confirm Password
        </label>
        <input
          name="confirmPassword"
          placeholder="Confirm your Password"
          required={true}
          type="text"
        />
      </div>
    </span>
    <button
      type="submit"
    >
      Register
    </button>
  </form>
</div>
`;
